src/css/style.css

```diff
@@ -0,0 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  background-color: var(--color-white);
+  border: 1px solid var(--border-color);
+  color: var(--color-dark);
+  padding: 32px 16px 16px;
```

Comment:
Styles for the card BEM block should be in card.css file

Reply:
ok